moto/neptune/models.py

```diff
@@ -244,6 +244,10 @@ def __init__(self, region_name: str, account_id: str):
         self.global_clusters: Dict[str, GlobalCluster] = dict()
         self._db_cluster_options: Optional[List[Dict[str, Any]]] = None
 
+    @property
+    def global_backend(self) -> "NeptuneBackend":
+        return neptune_backends[self.account_id]["us-east-1"]
```

Comment:
Without doing any research on these features, should this region be hard-coded?

Reply:
Neptune/RDS have a concept of global clusters, that are available everywhere. So you can create a global cluster in any region, and you can retrieve all clusters from any region as well.
Explicitly storing global state like this in `us-east-1` is the easiest way to deal with it IMO.